calico/felix/config.py

```diff
@@ -75,9 +75,23 @@ def __init__(self, config_path):
                                            "FelixHostname",
                                            socket.gethostname())
 
-        cfg_dict = fetcd.load_config(self.HOSTNAME,
-                                     self.ETCD_PORT)
+        self.STARTUP_CLEANUP_DELAY = 30
```

Comment:
Reworked this so that fetcd can call update_config once the config is ready rather than needing it to be ready beforehand.